src/legacy/core_plugins/console/np_ready/public/application/components/console_menu.tsx

```diff
@@ -103,8 +103,9 @@ export class ConsoleMenu extends Component<Props, State> {
 
   render() {
     const button = (
-      <EuiButtonIcon
-        iconType="wrench"
+      <button
```

Comment:
So In this particular case we are using button in a very specific way - this is actually bring the button inline with the play button. I'd rather not generalise/optimise too much for this case since it will very likely change in the future.

Reply:
Makes sense 👍